apps/docs/content/guides/database/full-text-search.mdx

```diff
@@ -751,6 +749,81 @@ When you want the search term to include a phrase or multiple words, you can con
 select * from search_books_by_title_prefix('Little+Puppy');
 ```
 
+## Web search syntax with `websearch_to_tsquery()` [#websearch-to-tsquery]
+
+The `websearch_to_tsquery()` function provides an intuitive search syntax similar to popular web search engines, making it ideal for user-facing search interfaces.
+
+### Basic usage
+
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="sql"
+  queryGroup="language"
+>
+<TabPanel id="sql" label="SQL">
+
+```sql
+select *
+from books
+where to_tsvector(description) @@ websearch_to_tsquery('english', 'green eggs');
+```
+
+</TabPanel>
+<TabPanel id="js" label="JavaScript">
+
+```js
+const { data, error } = await supabase
+  .from('books')
+  .select()
+  .textSearch('description', 'green eggs', { type: 'websearch' })
+```
+
+</TabPanel>
+</Tabs>
+
+### Quoted phrases
+
+Use quotes to search for exact phrases:
+
+```sql
+select * from books
+where to_tsvector(description || ' ' || title) @@ websearch_to_tsquery('english', '"Green Eggs"');
+-- Matches documents containing "Green" immediately followed by "Eggs"
+```
+
+### OR searches
+
+Use "or" (case-insensitive) to search for multiple terms:
+
+```sql
+select * from books
+where to_tsvector(description) @@ websearch_to_tsquery('english', 'puppy or rabbit');
+-- Matches documents containing either "puppy" OR "rabbit"
+```
+
+### Negation
+
+Use a dash (-) to exclude terms:
+
+```sql
+select * from books
+where to_tsvector(description) @@ websearch_to_tsquery('english', 'animal -rabbit');
+-- Matches documents containing "animal" but NOT "rabbit"
+```
+
+### Complex queries
+
+Combine multiple operators for sophisticated searches:
+
+```sql
+select * from books
+where to_tsvector(description || ' ' || title) @@
+  websearch_to_tsquery('english', '"Harry Potter" or "Dr. Seuss" -vegetables');
+-- Matches books by "Harry Potter" or "Dr. Seuss" but excludes those mentioning vegetables
+```
+
 ## Creating indexes
 
 Now that we have Full Text Search working, let's create an `index`. This will allow Postgres to "build" the documents preemptively so that they
```

Comment:
```suggestion
Now that you have Full Text Search working, create an `index`. This allows Postgres to "build" the documents preemptively so that they
```

Nothing to do with you, but while we're here let's fix some things :)